include/ibm/management_console_rest.hpp

```diff
@@ -533,14 +533,47 @@ inline void
             asyncResp->res.jsonValue["TransactionID"] = var;
             return;
         }
-        BMCWEB_LOG_DEBUG << "There is a conflict with the lock table";
-        asyncResp->res.result(boost::beast::http::status::conflict);
+
+        // Check if the session is active. If active, send back the lock
+        // conflict, else clear the stale entry in the lock table.
         auto var =
             std::get<std::pair<uint32_t, LockRequest>>(conflictStatus.second);
+        std::string sessionId = std::get<0>(var.second);
+        auto session =
+            persistent_data::SessionStore::getInstance().getSessionByUid(
+                sessionId);
```

Comment:
do we wanna do this for every conflicting request ?

Reply:
cant we do it just once in the constructor, so we remove stale entries in one shot when bmcweb comes up ? rather than checking them every time ?

Reply:
With the several defects data - what we saw is the locks are going stale while the bmcweb/bmc is running. Somehow the session from HMC is getting vanished with-out a proper cleanup path hitting at bmc. This is leaving stale locks. 
So we need a way to check if the conflicting lock is associated with any stale session OR and active session

Reply:
hmm, so this is ideally a bug in bmcweb where the sessions are removed in a path that we don't know (not logout/timeout) - this is essentially a workaround of that bug.